### ISSUE TITLE
UT-Midway

### ISSUE THREAD

**Issue by manoman31** [ZRiddle/PokeAlertsOnboarding#52]
This app is amazing.  If possible, please add resources in Midway/Heber UT.  Park city would rock too!!

**Comment by bellowsj** [comment]
@manoman31 - What team are you?

**Comment by manoman31** [comment]
Valor.  You?

**Comment by manoman31** [comment]
New to this...  Found the longitude and latitude

40.5365, -111.5008
40.4873, -111.4352

**Comment by bellowsj** [comment]
Also Valor, You might have seen me (jdb9393) and my wife (MidwayMtnMomma)
in a few gyms around town. What trainer level and screen name are you? Email me directly (bellowsj@gmail.com) and I might be able to share some info with you.

**Comment by manoman31** [comment]
Yes!  We finally found you!!!  Hahahaha....  So glad to finally meet!!  we know you guys very well.  We are mcbem, emcbem, meribericupcake, prplkat, ladefreakinda....  Our gyms stay up in Midway forever...  It's awesome!!
